ui/objects/semantic-ui/js/objects/editform/EditObjectForm.jsx

```diff
@@ -16,14 +16,15 @@ import { PartsInfo } from "./components/PartsInfo";
 
 export const EditObjectForm = () => {
   let { record } = useFormConfig();
+  console.log(record)
```

Comment:
Remove this debug print before we merge it (or you can use FormikStateLogger component from oarepo-ui that behaves well with dev/prod)